### PR TITLE
A0-824 Increased sudo account money. Allow unsafe RPC port. Changes required to run payout_stakers_bench

### PR DESCRIPTION
**Pull request by Marcin-Radecki**

### CODE REVIEW

**Review by fbielejec — APPROVED** [review]

File: utils.py [hunk 1]
```diff
@@ -360,7 +360,7 @@ def bootstrap_chain(account_ids, chain, benchmark_config=None, rich_accounts=[],
         1, 'gen', 'accounts/sudo_sk', 'accounts/sudo_aid')[0]
     chainspec['genesis']['runtime']['sudo']['key'] = sudo
     chainspec['genesis']['runtime']['balances']['balances'].append(
-        (sudo, 10 ** 17))
+        (sudo, 10 ** 20))
```
Comment (kostekIV) [inline utils.py:363]: optional: extract to the const

File: fabfile.py [hunk 2]
```diff
@@ -223,6 +224,10 @@ def install_prometheus(conn):
         'tar xvfz prometheus*.tar.gz'
     conn.run(install_cmd)
 
+@task
+def kill_nodes(conn):
+    kill_cmd = f'killall aleph-node'
+    conn.run(kill_cmd)
```
Comment (kostekIV) [inline fabfile.py:230]: no need for `f` and I would inline the kill_cmd into run function

**Review by kostekIV — APPROVED** [review]

File: fabfile.py (hunk 2 above)
Comment (Marcin-Radecki) [reply fabfile.py:230]: Good point

File: utils.py (hunk 1 above)
Comment (Marcin-Radecki) [reply utils.py:363]: Will do

**Pull request merged**